curriculum/challenges/english/25-front-end-development/workshop-library-manager/67116d7584d0b469b14579c2.md

```diff
@@ -0,0 +1,124 @@
+---
+id: 67116d7584d0b469b14579c2
+title: Step 5
+challengeType: 1
+dashedName: step-5
+---
+
+# --description--
+
+To display the book summaries, you can use the `about` property of each book.
+
+Create a `getBookSummaries` function with a `catalog` parameter. The `getBookSummaries` function should return an array containing all the `about` property value of each book.
+
+# --hints--
+
+You should create a `getBookSummaries` function.
+
+```js
+assert.isFunction(getBookSummaries)
+```
+
+Your `getBookSummaries` function should have a `catalog` parameter.
+
+```js
+assert.match(getBookSummaries.toString(), /catalog/)
+```
+
+Your `getBookSummaries` function should return an array.
+
+```js
+assert.isArray(getBookSummaries(library))
+```
+
+Your `getBookSummaries` function should use an array method.
```

Comment:
same comment here as earlier. This hint will need to be more specific since we are looking for high order functions instead of any array method like push or pop